docs/source/supergraphs.md

```diff
@@ -14,7 +14,7 @@ graph BT;
   gateway --- serviceA & serviceB & serviceC;
 ```
 
-Rover commands that interact with supergraphs begin with `rover supergraph`. These commands primarily deal with composition of a [supergraph schema](/federation/federated-types/overview/) that adheres to the [core schema specification](https://specs.apollo.dev/core/v0.1/).
+Rover commands that interact with supergraphs begin with `rover supergraph`. These commands primarily deal with [supergraph schemas](/federation/federated-types/overview/). These schemas adheres to the [core schema specification](https://specs.apollo.dev/core/v0.2/) instead of the GraphQL specification.
```

Comment:
```suggestion
Rover commands that interact with supergraphs begin with `rover supergraph`. These commands primarily deal with [supergraph schemas](/federation/federated-types/overview/), which adhere to the [core schema specification](https://specs.apollo.dev/core/v0.2/) in addition to the GraphQL specification.
```

... right? Supergraph schemas don't violate the main GraphQL spec or anything, do they?

Reply:
uhhhh you're probably right here. i was under the assumption that it was different because of `rover graph fetch` and `rover supergraph fetch`, but you're likely right here and federation is mostly just a standard implementation of a set of directives? as usual, writing the docs highlights the fact that my understanding is fuzzier than i realized.